src/components/organism/Balances.tsx

```diff
@@ -0,0 +1,128 @@
+  const [monthChosen, setMonthChosen] = React.useState<moment.Moment>(moment());
+  const [entries, setEntries] = React.useState<{[s: string]: Array<IBalanceEntry>}>({});
+
+  const createEntries = (month: IMonth) => {
+
+    const monthBalances: { [s: string]: {income: number, expense: number, liquidity: number}; } = {};
+    const liquidities = month.balance.sort((a, b) => {
+      if (a.date <= b.date ) {
+        return -1;
+      }
+      return 1;
+    });
+    monthBalances[`${month.year}-${month.month < 9 ? '0' : ''}${month.month}-01`] = {
+      expense: 0,
+      income: 0,
+      liquidity: month.start_balance,
+    };
+
+    liquidities.forEach(b => {
+      monthBalances[b.date] = {
+        expense: 0,
+        income: 0,
+        liquidity: b.money,
+      };
+    });
+
+    month.transactions.forEach(t => {
+      // TransactionType with capital first letter does not reflect enums from API with lowercase..
+      if (t.type.charAt(0).toUpperCase() + t.type.slice(1) === TransactionType.income) {
+        monthBalances[t.date].income += t.money;
+      } else {
+        monthBalances[t.date].expense += t.money;
+      }
+    });
+
+    const balanceEntries: Array<IBalanceEntry> = [];
+    Object.keys(monthBalances).forEach(be => {
+      balanceEntries.push({
+        date: be,
+        expense: monthBalances[be].expense,
+        income: monthBalances[be].income,
+        liquidity: monthBalances[be].liquidity});
+    });
+    return balanceEntries;
+  };
+
+  React.useEffect(() => {
+
+    // Fetch month and create balance entries for chosen month if not done already.
+    if (!(`${monthChosen.month()}-${monthChosen.year()}` in entries)) {
+      BalancesAPI.getMonth(monthChosen.month() + 1, monthChosen.year(), companyId)
+        .then(balanceEntries => {
+          if (balanceEntries.length !== 1) {
+            const newEntries = {...entries};
+            newEntries[`${monthChosen.month()}-${monthChosen.year()}`] = [];
+            setEntries(newEntries);
+          } else {
+            const newEntries = {...entries};
+            newEntries[`${monthChosen.month()}-${monthChosen.year()}`] = createEntries(balanceEntries[0]);
+            setEntries(newEntries);
+          }
+        })
+        .catch(error => {
+          alert(error);
+        });
+    }
+  }, [monthChosen]);
+
+  const entriesIndex = `${monthChosen.month()}-${monthChosen.year()}`;
+
+  return (
+    <div className={props.className}>
+      <div>
+        <div className={'title'}>
+          <h1>Balances</h1>
+          <h5>Showing income, expense and liquidity for a month</h5>
+        </div>
+        <MonthPicker month={monthChosen} setState={setMonthChosen} />
+        <BalancesTable entries={entries[entriesIndex] ? entries[entriesIndex] : []} />
+      </div>
+    </div>
+  );
+};
+
+export default styled(Balances)`
+  margin: 2em;
+
+  h1 {
+    font-weight: 700;
+    font-size: 1.8em;
+  }
+
+  h5 {
+    font-weight: 300;
+    line-height: .7em;
+  }
```

Comment:
Standard global styling values for typography is defined in `src/styling/global.ts`. You should use those or we should modify the typography global styling. Maybe another approach with a `<Typography variant={}>` component (like MaterialUI uses) is better. Idk. 🤷‍♂